code/datums/components/storage/concrete/stack.dm

```diff
@@ -39,7 +39,7 @@
 			_S.add(can_insert)
 			S.use(can_insert, TRUE)
 			return TRUE
-	return ..(S.change_stack(null, can_insert), override)
+	return ..(S.split_stack(null, can_insert), override)
```

Comment:
```suggestion
	I = S.split_stack(null, can_insert)
	return ..()
```
Changed args are automatically passed down correctly.

This is really bad practice, to alter the value that goes down the line, but eh, it was already like this.